content/docs/user-interface.md

```diff
@@ -0,0 +1,121 @@
+
+The first tab to be selected when you start the Document Switcher depends on the **[Settings > Preferences > MISC](../preferences/#misc) > Document Switcher > Enable MRU behavior** checkbox: When not checkmarked, the first-selected tab will be the tab to the right of the currently-active tab (or the first tab in the View, if the currently-active tab is the last in the View); in this mode, the order of the tabs in the Document Switcher is the same as the order of the tabs in the tab bar.  When that setting is checkmarked, the first-selected tab will be the tab that was active just before the current tab was activated (the "most recently used" tab); in this mode, the order of the tabs in the Document Switcher is the "most recently used" order, with the most recent at the top of the list, and the least recent at the bottom.
+
+When cycling through the tabs in the Document Switcher, if the bottom entry in the list is reached and you again press and release <kbd>Tab</kbd>, the selection will move to the top of the list (that is, the bold highlight will move to the top of the list); or, if the list was too long to entirely be displayed, the <kbd>Tab</kbd> will scroll the Document Switcher, so the next entry down will be made visible and will be selected (displayed as bold), and the entry that was previous at the top of the list will scroll out of the visible popup.
+
+If you hold down <kbd>Shift</kbd> (while still holding the <kbd>Ctrl</kbd> key to keep you in Document Switcher mode), and then press and release the <kbd>Tab</kbd>, it will cause the selection (bold highlight) to move to the previous entry in the tab list, rather than the next.  (And in the long-list scenario, it would scroll to the next tab above the top in the list, making that visible and scrolling the bottom tab in the list out of the visible popup.)
+
+The Document Switcher functionality can also be achieved using just the mouse (if you have a scroll wheel): Right-click in the editing area for a tab and hold the right mouse button, then begin scrolling the mouse wheel (in either direction) to display the Document Switcher popup; further scrolling of the scroll wheel will change which tab is selected in the list.  Releasing the right mouse button will cause the currently-selected tab to be activated in the current View.
```

Comment:
Additional mouse functionality:  while still holding down the right button, if you move the mouse and point to an entry somewhere else in the Doc Switcher list, and then left-click, the clicked-on will briefly be bolded, Doc Switcher window will close, and then the clicked-entry's tab will become the active tab.

Reply:
Confirmed and updated